components/wpt-results.html

```diff
@@ -0,0 +1,107 @@
+<!--
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="/bower_components/polymer/polymer-element.html">
+<link rel="import" href="/bower_components/polymer/lib/elements/dom-repeat.html">
+
+<dom-module id="wpt-results">
+  <template>
+    <style>
+      :host {
+        display: block;
+      }
+      .WPTResults__spec {
```

Comment:
It's a naming convention called [BEM](http://getbem.com/naming/). I picked it up when pairing with a designer and it sort of stuck. However, with web components and specifically shadow DOM, BEM's benefits of component reusability pretty much don't matter. Because of this, I'll remove the prefixes from these classes (so it's just `.spec`).